README.md

```diff
@@ -73,13 +74,40 @@ To use it during training :
 ```
 
 Details about args:
-- `optimizer`: optimizer needed for training the model ( SGD/Adam )
-- `peak_lr`: the peak learning required for explore phase to escape narrow minimas
-- `warmup_steps`: steps required for warmup( usually needed for adam optimizers/ large batch training) Default value: 0
-- `explore_steps`: total steps for explore phase.
-- `total_steps`: total training budget steps for training the model
-
-
+- `optimizer`: Optimizer needed for training the model ( SGD and Adam has compatibility with cycle_momentum).
+- `max_lr` (float): The peak learning required for explore phase to escape narrow minimums.
+- `total_steps` (int): The total number of steps in the cycle. Note that if a value is not provided here, 
+  then it must be inferred by providing a value for epochs and steps_per_epoch.
+- `epochs` (int): The number of epochs to train for. This is used along with steps_per_epoch in order to infer 
+  the total number of steps in the cycle if a value for total_steps is not provided.
+- `steps_per_epoch` (int): The number of steps per epoch to train for. This is used along with epochs in order to 
+  infer the total number of steps in the cycle if a value for total_steps is not provided.
+- `pct_start` (float): The percentage of the total steps spent increasing the learning rate.
+- `pct_explore` (float): The percentage of the total steps spent on explore phase (keeping max the learning rate).
+- `cycle_momentum` (bool): If ``True``, momentum is cycled inversely
```

Comment:
not sure if we want to provide cyclical momentum . No idea how it will affect the explore phase and the effect it may have on reaching a wider minima

Reply:
So, it will be better to set this parameter `False` by default